backend/src/main/java/com/timlenny/backend/controller/TopicController.java

```diff
@@ -28,6 +28,11 @@ public Topic addTopic(@RequestBody TopicDTO topicToAddDTO) {
         return topicService.addTopic(topicToAddDTO);
     }
 
+    @PutMapping("/topic/positions")
```

Comment:
@RequestMapping("/api/topic")you can write this on line 13 instead of adding it each time.